sysdig/internal/client/v2/agentaccesskey.go

```diff
@@ -0,0 +1,122 @@
+package v2
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const (
+	GetAgentAccessKeyByIdPath = "%s/platform/v1/access-keys/%s"
+	CreateAgentAccessKeyPath  = "%s/platform/v1/access-keys"
+	DeleteAgentAccessKeyPath  = "%s/platform/v1/access-keys/%s"
+	PutAgentAccessKeyPath     = "%s/platform/v1/access-keys/%s"
+)
+
+type AgentAccessKeyInterface interface {
+	Base
+	GetAgentAccessKeyById(ctx context.Context, id string) (*AgentAccessKey, error)
```

Comment:
by the go styleguide rename `Id` to `ID`: 
```Words in names that are initialisms or acronyms (e.g., URL and NATO) should have the same case. URL should appear as URL or url (as in urlPony, or URLPony), never as Url. This also applies to ID when it is short for “identifier”; write appID instead of appId.```

Reply:
Nitpicking but fix in other places in this file :)